kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -1341,7 +1341,7 @@ public void apply(KToken k) {
                 convert(k.sort(), false);
                 sb.append("}(");
                 if (module.sortAttributesFor().get(k.sort()).getOrElse(() -> Att.empty()).getOptional("hook").orElse("").equals("STRING.String")) {
-                    sb.append(k.s());
+                    sb.append(StringUtil.enquoteKString(k.s()));
```

Comment:
uh, this is seriously going to break the llvm backend. The kore representation for the String
```
foo
bar
```

should be "foo\nbar", not "\"foo\\nbar\"".

Or am I misunderstanding what this is doing?

Reply:
So, right now the k file contains domain values like
```
\dv{??}("foo
bar").
```

I am attempting to convince it to represent them as
```\dv{??}("foo\nbar")
```